frontend/resourceadm/utils/resourceListUtils/resourceListUtils.test.ts

```diff
@@ -0,0 +1,9 @@
+import { dataFilteredResultMock, dataToFilterMock, searchValueMock } from "resourceadm/data-mocks/resourceListMocks";
+import { filterTableData } from "./resourceListUtils";
+
+describe('filterTableData', () => {
+  it('Filter out the data searched for', () => {
+    const result = filterTableData(searchValueMock, dataToFilterMock);
+    expect(result).toEqual(dataFilteredResultMock);
```

Comment:
The code is good, but for unit testing, it's better not to use general mocks as expected outcomes. This can make the test less clear about the actual expected string or object. If someone else uses the same mock in a different test, it might lead to issues if we modify the mock later.

Reply:
We should also make multiple test cases, where the search string is lowercased, UPPERCASED, KebabCased and maybe even with words with spacing. This ensures that we get matches regardless of the casing. Then we should test this function with search string that has a match and one without a match.

Reply:
When testing utility-methods we need to ensure they are behaving as expected in many cases. :)

Reply:
I fix ;)